client/X11/xf_graphics.c

```diff
@@ -438,7 +438,7 @@ static BOOL xf_Pointer_New(rdpContext* context, rdpPointer* pointer)
 
 #endif
 fail:
-	WLog_DBG(TAG, "%s: %ld", __func__, rc ? pointer : -1);
+	WLog_DBG(TAG, "%s: %ld", __func__, rc ? pointer : NULL);
```

Comment:
you might take another look at the change on master, the format string is wrong.

Reply:
Ah, I was looking at the wrong change. 2da280b8a1748052b70b3f5a1ef0d8e932c33adc is the relevant commit.

Reply:
you copied it wrong ;)